dace/codegen/targets/cuda.py

```diff
@@ -208,7 +208,7 @@ def preprocess(self, sdfg: SDFG) -> None:
             if (isinstance(node, nodes.MapEntry)
                     and node.map.schedule in (dtypes.ScheduleType.GPU_Device, dtypes.ScheduleType.GPU_Persistent)):
                 if state.parent not in shared_transients:
-                    shared_transients[state.parent] = state.parent.shared_transients()   
+                    shared_transients[state.parent] = state.parent.shared_transients()
```

Comment:
Please separate content  updates from stylistic ones in future, makes reviewing more difficult